cmd/codegen/generator/go/templates/src/dagger.gen.go.tmpl

```diff
@@ -173,3 +173,14 @@ func (e *ExecError) Message() string {
 func (e *ExecError) Unwrap() error {
 	return e.original
 }
+
+{{ range .Types }}
+{{ if eq .Kind "SCALAR" }}{{ template "_dagger.gen.go/scalar.go.tmpl" . }}{{ end }}
+{{ if eq .Kind "OBJECT" }}{{ template "_dagger.gen.go/object.go.tmpl" . }}{{ end }}
+{{ if eq .Kind "INPUT_OBJECT" }}{{ template "_dagger.gen.go/input.go.tmpl" . }}{{ end }}
+{{ if eq .Kind "ENUM" }}{{ template "_dagger.gen.go/enum.go.tmpl" . }}{{ end }}
+{{ end }}
+
+{{ if IsModuleCode }}
+{{ template "_dagger.gen.go/module.go.tmpl" $.Schema }}
+{{ end }}
```

Comment:
This is really what this patch is all about :tada:

We move the hard-coded visitor pattern from the caller into here, so that we can customize exactly how we want to do this.

In practice, for other files in the future (like the sub-package), we might not need to define different paths for each kind, we might just want to get the names, etc.